packages/block-editor/src/components/font-sizes/test/fluid-utils.js

```diff
@@ -67,91 +74,105 @@ describe( 'getComputedFluidTypographyValue()', () => {
 		);
 	} );
 
-	describe( 'getTypographyValueAndUnit', () => {
-		it( 'should return the expected return values', () => {
-			[
-				{
-					value: null,
-					expected: null,
-				},
-				{
-					value: false,
-					expected: null,
-				},
-				{
-					value: true,
-					expected: null,
-				},
-				{
-					value: [ '10' ],
-					expected: null,
-				},
-				{
-					value: '10vh',
-					expected: null,
-				},
-				{
-					value: 'calc(2 * 10px)',
-					expected: null,
-				},
-				{
-					value: 'clamp(15px, 0.9375rem + ((1vw - 7.68px) * 5.409), 60px)',
-					expected: null,
-				},
-				{
-					value: '10',
-					expected: {
-						value: 10,
-						unit: 'px',
-					},
+	it( 'should return default font size when maximumViewPortWidth not a supported value or unit', () => {
```

Comment:
```suggestion
	it( 'should return null when maximumViewPortWidth is not a supported value or unit', () => {
```

Nit: should this test say `should return null`? For the behaviour we're after, `getTypographyFontSizeValue` will return the default font size, but it internally uses `getComputedFluidTypographyValue`, which will return `null`, and `getComputedFluidTypographyValue` will be responsible for returning the default value.

Just wanted to make sure I'm not missing anything, in case the intention here was different!

Reply:
Good catch. Copy pasta 🍝 

Thanks!